client.go

```diff
@@ -250,14 +286,13 @@ func (c *Client) setDefaults() {
 		Consumer:  "",
 		AutoAck:   true,
 		Exclusive: true,
-		NoLocal:   false,
```

Comment:
Well, `NoLocal` has no effect since it only ensures that a consumer cannot consume messages published over the same connection. But we have one connection for when we publish and one connection for when we consume.